test/fixtures/files/plugins.js

```diff
@@ -0,0 +1,9 @@
+module.exports = [{
+  func: () => {},
+  description: 'Test action #1',
+  name: 'test',
+}, {
+  func: () => {},
+  description: 'Test action #2',
+  name: 'test2',
+}];
```

Comment:
Can we add one test-spec for single non-array object being exported? Just like `rnpm-link` does for now.